server/configuration.go

```diff
@@ -46,8 +42,8 @@ type configuration struct {
 	SyntheticUserAuthService           string `json:"syntheticUserAuthService"`
 	SyntheticUserAuthData              string `json:"syntheticUserAuthData"`
 	AutomaticallyPromoteSyntheticUsers bool   `json:"automaticallyPromoteSyntheticUsers"`
-	DisableSyncMsg                     bool   `json:"disableSyncMsg"`
-	DisableCheckCredentials            bool   `json:"disableCheckCredentials"`
+	UseSharedChannels                  bool   `json:"experimentalUseSharedChannels"`
```

Comment:
Curious as to the word choice "Use". `UseSharedChannels` makes me think this is related to the SharedChannels architecture. (`EnableSharedChannels`). `ShareChannels` or `AllowSharedChannels` seems more appropriate. I'm 0/5 and may be misunderstanding.

Reply:
@sbishel, this is indeed referring to the shared channels architecture feature that won't be productized for v10. Agree the wording is confusing -- is there another term that might help clarify? Maybe even explicitly `UseSharedChannelsArchitecture` or `EnableSharedChannelsArchitecture`?

Reply:
@sbishel, just circling back to our conversation above -- any thoughts after the clarification?

Reply:
If it is referring to the architecture, I think the current naming is fine, no need to add `Architecture`. Its use in `plugin.go` made me think it was more of an on/off flag rather than which architecture to use. My misunderstanding of its full purpose...only looking at the PR.